kinto/schema_validation.py

```diff
@@ -130,6 +130,8 @@ def validate_from_bucket_schema_or_400(data, resource_name, request, ignore_fiel
 
     # Validate or fail with 400.
     schema = bucket[metadata_field]
+    if id_field not in schema.get("properties", {}):
+        ignore_fields += (id_field,)
```

Comment:
Maybe we can move this logic into `validate_schema` and not have to duplicate it in `kinto/views/records.py`.